src/main/java/ua/lviv/iot/algo/part1/lab1/repository/HornetRepository.java

```diff
@@ -0,0 +1,34 @@
+package ua.lviv.iot.algo.part1.lab1.repository;
+
+import org.springframework.stereotype.Repository;
+import ua.lviv.iot.algo.part1.lab1.models.Hornet;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Repository
```

Comment:
Please check which classes should be marked with @Repository
the map should be used in class with @Service annotation